src/Data/Fin/Subset/Properties.agda

```diff
@@ -59,6 +67,51 @@ module _ {n} {p q : Subset n} where
   in⊆in-⇔ : p ⊆ q ⇔ inside ∷ p ⊆ inside ∷ q
   in⊆in-⇔ = equivalence in⊆in drop-∷-⊆
 
+  s⊆s : ∀ {s} →  p ⊆ q → s ∷ p ⊆ s ∷ q
+  s⊆s {.inside} p⊆q here = here
+  s⊆s {s} p⊆q (there ∈p) = there (p⊆q ∈p)
+
+  out⊂ : ∀ {y} → p ⊂ q → outside ∷ p ⊂ y ∷ q
+  out⊂ (p⊆q , x , x∈q , x∉p) = out⊆ p⊆q , suc x , there x∈q , λ {(there x∈p) → contradiction x∈p x∉p}
+
+  out⊂in : p ⊆ q → outside ∷ p ⊂ inside ∷ q
+  out⊂in p⊆q = out⊆ p⊆q , zero , here , λ ()
+
+  out⊂in-⇔ : p ⊆ q ⇔ outside ∷ p ⊂ inside ∷ q
+  out⊂in-⇔ = equivalence out⊂in (drop-∷-⊆ ∘ proj₁)
+
+  out⊂out-⇔ : p ⊂ q ⇔ outside ∷ p ⊂ outside ∷ q
+  out⊂out-⇔ = equivalence out⊂ drop-∷-⊂
+
+  in⊂in : p ⊂ q → inside ∷ p ⊂ inside ∷ q
+  in⊂in (p⊆q , x , x∈q , x∉p) = in⊆in p⊆q , suc x , there x∈q , λ {(there x∈p) → contradiction x∈p x∉p}
+
+  in⊂in-⇔ : p ⊂ q ⇔ inside ∷ p ⊂ inside ∷ q
+  in⊂in-⇔ = equivalence in⊂in drop-∷-⊂
+
+drop-∷-⊂′ : ∀ {n s} {p₁ p₂ : Subset n} → s ∷ p₁ ⊂′ s ∷ p₂ → p₁ ⊂′ p₂
+drop-∷-⊂′ (sp₁⊆sp₂ , sp₂⊈sp₁) = drop-∷-⊆ sp₁⊆sp₂ , λ p₂⊂p₁ → sp₂⊈sp₁ (s⊆s p₂⊂p₁)
```

Comment:
Again can be simplified to ` sp₂⊈sp₁ ∘ s⊆s`

Reply:
Done.